subsys/bluetooth/host/keys.c

```diff
@@ -51,7 +59,24 @@ struct bt_keys *bt_keys_get_addr(u8_t id, const bt_addr_le_t *addr)
 			first_free_slot = i;
 		}
 	}
-
+#if defined(CONFIG_BT_KEYS_OVERWRITE_OLDEST)
+	if (first_free_slot == ARRAY_SIZE(key_pool)) {
+		size_t oldest_slot = 0;
+
+		for (i = 1; i < ARRAY_SIZE(key_pool); i++) {
+			if (key_pool[oldest_slot].aging_counter
+			    > key_pool[i].aging_counter) {
+				oldest_slot = i;
+			}
+		}
+		bt_unpair(key_pool[oldest_slot].id,
+			  &key_pool[oldest_slot].addr);
```

Comment:
Empty line here. Considering how many times I see you use `key_pool[oldest_slot]` it becomes pretty clear that a helper variable `struct bt_keys *oldest = &key_pool[oldest_slot]` would make the code more readable and avoid overlong lines.